bgpd/bgpd.h

```diff
@@ -1047,6 +1049,7 @@ struct peer {
 	struct stream_fifo *ibuf; // packets waiting to be processed
 	struct stream_fifo *obuf; // packets waiting to be written
 
+	/* used as a block to deposit raw wire data to */
```

Comment:
Hate to ask that, but is this needed? :)

Reply:
Nope, leftovers from another commit, good catch